src/dir.targets

```diff
@@ -10,6 +10,15 @@
   <Target Name="GetAssemblyVersion"
           Returns="$(AssemblyVersion)"/>
 
+  <!-- These gets overwritten when we restore the build tools and import the below targets -->
+  <Target Name="Build" Condition="!Exists('$(ToolsDir)Build.Common.targets')" >
+    <CallTarget Targets="_RestoreBuildTools" />
+  </Target>
+  <!-- Set a default clean target -->
+  <Target Name="Clean">
```

Comment:
Thanks for the explanation @chcosta.

I think everything makes sense to me except for the default behavior of Clean which is to delete BinDir. In my opinion that big hammer only makes sense for the root level build and so I wouldn't do that for leave node projects. So perhaps and empty Clean target is good enough here.